requirements-dev.txt

```diff
@@ -1,11 +1,13 @@
 git+https://github.com/asdf-format/asdf
 git+https://github.com/asdf-format/asdf-transform-schemas
 git+https://github.com/asdf-format/asdf-wcs-schemas
-git+https://github.com/astropy/astropy
+astropy>=0.0.dev0
 git+https://github.com/astropy/asdf-astropy
 git+https://github.com/spacetelescope/crds
 git+https://github.com/spacetelescope/gwcs
+numpy>=0.0.dev0
```

Comment:
This change does absolutely nothing in the context of the rest of the changes here. In fact, one can see in the CI: https://github.com/spacetelescope/romancal/actions/runs/4926111203/jobs/8801225303#step:10:265 that it is using `numpy 1.24.3` which is the latest release version of numpy NOT some "nightly-build".

If you want to use this to actually access the nightly builds (which in reality are more like weekly builds), you need to specify an alternate index for `pip` to search.

For example https://github.com/astropy/astropy/blob/337798f36c9aa2ae98190c5819bde758d3aa5a40/tox.ini#L27 specifies the "nightly-build" provided by anaconda.